### PR TITLE
Dmp 2831.v67 v68 v69 data model

### PR DESCRIPTION
**Pull request by cmagowan-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/common/entity/ObjectAdminActionEntity.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+@EqualsAndHashCode(callSuper = false)
+public class ObjectAdminActionEntity {
+
+    @Id
+    @Column(name = "oaa_id")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "oaa_gen")
+    @SequenceGenerator(name = "oaa_gen", sequenceName = "oaa_seq", allocationSize = 1)
+    private Integer id;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = false)
```
Comment (johnrace0) [inline src/main/java/uk/gov/hmcts/darts/common/entity/ObjectAdminActionEntity.java:32]: just checking - do these definitely need to be EAGER?

File: src/main/resources/db/migration/postgres/V1_298_2__v67_v68_v69_database_updates.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+CREATE INDEX usr_upea_idx       ON USER_ACCOUNT(UPPER(user_email_address));
+CREATE INDEX usr_ag_idx         ON USER_ACCOUNT(account_guid)             ;
```
Comment (johnrace0) [inline src/main/resources/db/migration/postgres/V1_298_2__v67_v68_v69_database_updates.sql:2]: not that it makes a difference but spaces before semi-colon looks strange

File: src/main/java/uk/gov/hmcts/darts/common/entity/ObjectAdminActionEntity.java (hunk 1 above)
Comment (cmagowan-hmcts) [reply src/main/java/uk/gov/hmcts/darts/common/entity/ObjectAdminActionEntity.java]: done

File: src/main/java/uk/gov/hmcts/darts/common/entity/CourtCaseEntity.java [hunk 3]
```diff
@@ -120,6 +116,15 @@ public class CourtCaseEntity extends CreatedModifiedBaseEntity {
...
     private List<JudgeEntity> judges = new ArrayList<>();
 
+    @Column(name = "is_data_anonymised")
+    private boolean isDataAnonymised;
+
+    @Column(name = "data_anonymised_by")
+    private Integer dataAnonymisedBy;
+
+    @Column(name = "data_anonymised_ts")
+    private OffsetDateTime dataAnonymisedTs;
```
[the remark this answers is not in the document]
Comment (cmagowan-hmcts) [reply src/main/java/uk/gov/hmcts/darts/common/entity/CourtCaseEntity.java:126]: done

**Review by johnrace0 — APPROVED** [review]

**Pull request merged**